tests/integration/test_spans.py

```diff
@@ -308,8 +308,8 @@ def envelope_with_spans(
                     {
                         "description": "https://example.com/p/blah.js",
                         "op": "resource.script",
-                        "span_id": "bd429c44b67a3eb1",
-                        "segment_id": "968cff94913ebb07",
+                        "span_id": "b0429c44b67a3eb1",
+                        "segment_id": "b0429c44b67a3eb1",
```

Comment:
I gave this span a new span_id and the same segment_id, to make sure it is classified as `is_segment`.

Reply:
Also please put that as a comment in the code

Reply:
Ah already merged, dw about it. I'll throw up a PR.

Reply:
#3453